### ISSUE TITLE
Common means to dereference key references

### ISSUE THREAD

**Issue by jricher** [ietf-wg-gnap/gnap-core-protocol#154]
> I disagree: when used between the Client and AS this is clearly required for interop.

_Originally posted by @yaronf in https://github.com/ietf-wg-gnap/gnap-core-protocol/pull/152#discussion_r548538220_

**Comment by fimbault** [comment]
In general I would tend to think this doesn't need to be specified by us, supporting the current text "The means of dereferencing this value are out of scope for this specification."

But before we can conclude, I'd like to ask a few questions: 
- related the lifecycle of the referenced key (cf #161) : is it something ephemeral (linked to a specific request) or not ? (but knowing that continuation could last for some time, what do we do then?) 
- could it have an impact on error management? (for instance collision of references?)
